main.js

```diff
@@ -0,0 +1,204 @@
+  let currentTimeData = convertTime(currentDay.dt).toObject();
+  return {
+    day: dayForecasted,
+    weather: weather,
+    condition: condition,
+    icon: 'http://openweathermap.org/img/w/' + iconString + '.png',
+    timeData: currentTimeData
+  }
+
+}
+
+const getDaysArray = (data, days) => {
+  let daysObj = {
+    // Do not return a 6th day
+    // In some cases we get a 6th array
+  }
+  let start = 0;
+  let eachDayArray = []
+  for (let index = 0; index < data.length; index++) {
+    let currentDay = days[start];
+    const forecastObj = data[index];
+    if (_.contains(forecastObj, currentDay)) {
+      // console.log('Add to array');
+      eachDayArray.push(forecastObj)
+    } else {
+      // add eachDayArray to the daysObj to keep track of forecast days
+      daysObj[`day${start}`] = eachDayArray;
+      // empty array for the next day's data
+      eachDayArray = [];
+      // add object containing second day to array 
+      start += 1; // add one to start to keep track of days
+      eachDayArray.push(forecastObj)
+    }
+  }
+  // add last array to our daysObj 
+  daysObj[`day${start}`] = eachDayArray;
+  // Get daysObj to only get five days 
+  var size = Object.keys(daysObj).length;
+  // delete if in the request we get 6 days 
+  if (size >= 6) {
+    delete daysObj.day5;
+  }
+  // console.log(daysObj);
+  return daysObj
+
+}
+
+
+
+const renderWeather = () => {
+  $('.data').empty();
+  // create HTML and append to .data
+  let source = $("#weather-template").html();
+  let template = Handlebars.compile(source);
+  let newHTML = template(weatherData[0]);
+
+  $('.data').append(newHTML);
+};
+
+// console.log(weekData);
+const renderWeekWeather = () => {
+  $('.forecast').empty();
+  for (const key in weekData) {
+    if (weekData.hasOwnProperty(key)) {
+      const element = weekData[key];
+      let middle = Math.floor(element.length / 2);
+
+      // create HTML and append to .data
+      let source = $("#forecast-template").html();
+      let template = Handlebars.compile(source);
+      let newHTML = template(element[middle]);
+
+      $('.forecast').append(newHTML);
+    }
+  }
+};
+// Borrowed this method to get a query.
+// stackoverflow.com/questions/486896/adding-a-parameter-to-the-url-with-javascript
+const insertParam2 = (key, value) => {
+  key = encodeURIComponent(key);
+  value = encodeURIComponent(value);
+
+  let s = document.location.search;
+  let kvp = key + "=" + value;
+
+  let r = new RegExp("(&|\\?)" + key + "=[^\&]*");
+
+  s = s.replace(r, "$1" + kvp);
+
+  if (!RegExp.$1) {
+    s += (s.length > 0 ? '&' : '?') + kvp;
+  };
+
+  // document.location.search = s;
+
+  return s
+}
+
+let stateQuery = null;
+
+$('.search').on('click', () => {
+  // reset the data array;
+  weatherData = [];
+  let search = $('#search-query').val();
+  search = insertParam2('q', search);
+  // console.log(search);
+  stateQuery = search;
+
+  let startIndex = 0;
+  $('.weather-card').show();
+  fetch(search);
```

Comment:
I'm guessing `fetch` is for the current weather? Should be re-named accordingly to distinguish from `fetchFiveDay`